examples/with-styled-components/pages/_document.js

```diff
@@ -2,20 +2,19 @@ import Document, { Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
 
 export default class MyDocument extends Document {
-  static getInitialProps ({ renderPage }) {
```

Comment:
This looks like a stylistic change not related to this PR, probably caused by prettier?

Reply:
Yes, I use prettier.
Should I fix it as it was?